src/main/java/sirius/biz/jobs/params/SelectParameter.java

```diff
@@ -38,7 +37,7 @@ protected SelectParameter(String name, String label) {
      *
      * @return the parameter itself for fluent method calls
      */
-    public P withMultipleOptions() {
+    protected P withMultipleOptions() {
```

Comment:
This entire commit will later be reverted in commit 497737be45094b9f1cae1a827040cdfac81d8f8c of this PR and done properly. Sorry for this. Please see the [files changed](https://github.com/scireum/sirius-biz/pull/1763/files) for the effective changes on these files.